query-engine/driver-adapters/js/adapter-libsql/README.md

```diff
@@ -3,3 +3,93 @@
 Prisma driver adapter for Turso and libSQL.
 
 See https://prisma.io/turso for details.
+
+The following usage tutorial is valid for Prisma 5.4.2 and later versions.
+
+## How to install
+
+After [getting started with Turso](https://www.prisma.io/blog/prisma-turso-ea-support-rXGd_Tmy3UXX#create-a-database-on-turso), you can use the Turso serverless driver to connect to your database. You will need to install the `@prisma/adapter-libsql` driver adapter and the `@libsql/client` serverless driver.
+
+```sh
+npm install @prisma/adapter-libsql
+npm install @libsql/client
+```
+
+Make sure your Turso database connection string and authentication token is copied over to your `.env` file. The connection string will start with `libsql://`.
+
+```env
+# .env
+TURSO_AUTH_TOKEN="eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCJ9..."
+TURSO_DATABASE_URL="libsql://turso-prisma-random-user.turso.io"
+```
+
+You can now reference this environment variable in your `schema.prisma` datasource. Make sure you also include the `driverAdapters` Preview feature.
```

Comment:
I don't see any env var referenced in the datasource block of the schema.prisma, but instead a local URL.

Reply:
Thank you, this is a copy-paste error of me trying to fit [Turso snippets](https://www.prisma.io/blog/prisma-turso-ea-support-rXGd_Tmy3UXX) with the same structure used in [PlanetScale and Neon's blog post](https://www.prisma.io/blog/serverless-database-drivers-KML1ehXORxZV).